config/api.yml

```diff
@@ -1421,6 +1421,26 @@
       :get:
       - :name: read
         :identifier: event_streams_show
+  :event_sets:
```

Comment:
Why `event_set` as opposed to `event_definition_set`?  When I hear the former I think of a set of events like "vm state change events".

@agrare @gtanzillo opinions here?

Reply:
Yeah if the class name is MiqEventDefinitionSet I don't see why we wouldn't just call this event_definition_sets